build.gradle

```diff
@@ -200,6 +200,11 @@ dependencies {
     implementation group: 'com.github.wnameless.json', name: 'json-base', version: '2.2.1'
     implementation "com.fasterxml.jackson.core:jackson-databind:${jackson_version}"
     implementation "com.fasterxml.jackson.core:jackson-annotations:${jackson_version}"
+    // Forced dependency from json-schema-validator, we add nop to deactivate it
+    implementation group: 'org.slf4j', name: 'slf4j-api', version: '2.0.6'
+    implementation group: 'org.slf4j', name: 'slf4j-nop', version: '2.0.6'
```

Comment:
Is there some indirect dependency on `slfj` cause we usually use `log4j`

Reply:
Per the comment, it's a forced dependency from `json-schema-validator`.